steps/run/spec.schema.json

```diff
@@ -10,6 +10,16 @@
       "type": "object",
       "description": "Parameters to run the workflow with.",
       "additionalProperties": {"type": "string"}
+    },
+    "parameters_array": {
```

Comment:
I spent a bit of time thinking about this name after our discussion last week -- might I suggest calling this `parameterSets` instead (note most steps use camelCase)? I'm afraid it won't be clear here that each item of the parameters array will be a separate invocation, instead of just passing all the parameters to one run (even though that isn't technically possible).

Reply:
Done!